defaults/main.yml

```diff
@@ -5,4 +5,24 @@ rstudio_connect_version_short: "{{ rstudio_connect_version | regex_replace('.[^.
 rstudio_connect_install: []
 
 rstudio_connect_www_port: 3939
-rstudio_connect_www_address: 0.0.0.0
+rstudio_connect_config_override: ""
+
+rstudio_connect_licensing_override: |
+  [Licensing]
+  ExpirationUIWarning = false
+
+rstudio_connect_http_override: |
+  [HTTP]
+  ; RStudio Connect will listen on this network address for HTTP connections.
+  Listen = :{{ rstudio_connect_www_port }}
+
+rstudio_connect_authentication_override: |
+  [Authentication]
+  ; Specifies the type of user authentication.
+  Provider = "password"
+
+rstudio_connect_package_repos:
+  - name: CRAN
+    url: https://packagemanager.rstudio.com/cran/__linux__/bionic/latest
+  - name: RSPM
+    url: https://packagemanager.rstbudio.com/cran/__linux__/bionic/latest
```

Comment:
We set here bionic as source, but is this in requirements for only OS supported? maybe we should add this as variable? It's required to have binaries of packages built under same OS as connect is running.

Reply:
Ok, I see now that it's only for debian like systems, whole playbook is written for debian, but maybe we should be more strict here and say e.g. only bionic, focal not supported yet.

Reply:
@DamianBudelewski, the whole role currently only [supports](https://github.com/Appsilon/ansible-rstudio-connect/blob/main/meta/main.yml#L14) Focal (20.04) distribution. I'll change the behavior to build RSPM repository URLs based on `{{ ansible_lsb.codename }}`, as explained [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_vars_facts.html).